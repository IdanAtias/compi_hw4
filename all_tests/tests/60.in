F x;
switch(40 C){
	case 20 C: x = 2 F+3 C; break;
	case 30 F: print "42"; break;
	case 23 K: bool x; break;
}
