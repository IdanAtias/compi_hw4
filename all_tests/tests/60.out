line 5: variable x is already defined
